Vizzuality/landgriffon
# Rewrite the analysis' year filters

**⚠️ This PR will be merged in #143's branch, not `staging`.**

This PR rewrites the analysis' year filters to simplify the logic and it updates the query made to retrieve the years so that the filters are fully functional.

## Testing instructions

- Switch between the different layers and make sure the options of the years filters are updated and that the default selected value exists in the list of options
- Select a material or indicator, make sure the request to get the options contains both the material and indicator IDs
- Switch to the table view, make sure the component is displayed correctly
